src/StaticContentHandler.cpp

```diff
@@ -56,17 +56,17 @@ void StaticContentHandler::handleRequest(const HTTPRequest &request, HTTPRespons
 	std::string host = request.getHost();
 	// std::cout << "host in handleRequest: " << host << std::endl;
 
-	std::string path;
-
 	// for ease of use during deployment
 	// this if/else allows to reach target with tester or browser
-	if (host == "localhost:8080")
-		path = webRoot + requestTarget;
-	else
-		path = webRoot + "/" + host + requestTarget;
+
+	// if (host == "localhost:8080")
+	// 	path = webRoot + requestTarget;
+	// else
+	// 	path = webRoot + "/" + host + requestTarget;
```

Comment:
was this comment also in some other object?

Reply:
Yes, before I was doing that across 2 objects, I added a var _path to the request to simplify